app/views/users/registrations/edit.html.erb

```diff
@@ -4,29 +4,48 @@
   <%= f.error_notification %>
 
   <div class="form-inputs">
-    <%= f.input :email, required: true, autofocus: true %>
+    <p style="background-color:tomato; color:white"> Please make sure to enter your Current Password to make any Update </p>
```

Comment:
Id like to have the style in a css sheet. Lets move the style out of the html.

Reply:
I will create the CSS for that.